modules/persistent-storage-csi-sc-disable.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-disable_{context}"]
+= Disabling the default storage class
+
+Disabling the default storage class allows you to accomplish several different objectives:
+
+* Enforce static provisioning by disabling dynamic provisioning.
```

Comment:
Should these verbs be gerunds? Enforcing, preventing, Renaming? Seems to sound better to me.